docs/resources/firewalld.md.erb

```diff
@@ -0,0 +1,88 @@
+---
+title: About the firewalld Resource
+---
+
+# firewalld
+
+Use the `firewalld` InSpec audit resource to test rules set to match IP address's with hostnames.
+
+An firewalld rule specifies an IP address and what it's hostname is along with an aliase names it can have.
```

Comment:
`s/it's/its` and `s/aliase/alias`. Also, this looks like the description for the `etc_hosts` resource, not the `firewalld` resource.